### ISSUE TITLE
Multi Link Picker is missing QueryString in Model

### ISSUE THREAD

**Issue by AreYouSureYouKen** [umbraco/Umbraco-CMS#9423]
The current Multi Link Picker property editor separates the URL and QueryString/Anchor in two different input fields. If you copy paste the full url into the URL portion of the editor and save, it saves the full url into the URL Property and it works as expected on the website. However when you open it again, it loads the URL separated in the two editor fields.

When looking at the code, it gets saved in value.queryString. This is not a property available in the model however.
I have tested this by extending the current Link model and adding the string property QueryString and it works as expected.

## Umbraco version

I am seeing this issue on Umbraco version: 8.6.3 and 8.9.1, probably everything in between and before as well.

Reproduction
------------

### Bug summary

As stated above, QueryString/Anchor gets separated, but there is no separate property available in the Link model.

### Steps to reproduce

Add a Multi Link Picker property editor to a document type, add a link with a querystring or anchor by entering it into the URL portion, save.
It works as expected with the .Url property.
Edit the existing link, you see it is now separated into the URL and QueryString/Anchor portion, save again.
The QueryString/Anchor portion of the url is now missing.

It seems like an easy fix and I am willing to make a PR for this myself but if I'm not mistaken, I need a bug report and ticket number for that first, so that's what this is for.

**Comment by rasmusjp** [comment]
I did a quick test in 8.9.1 and I'm not able to reproduce it, if I paste in an URL with a querystring in the Link field and save+publish the url with querystring is shown, if I then edit the link it's correctly split into Link and Anchor / querystring fields, and if I save+publish again i still see the url with querystring rendered on the frontend

Note that the MultiUrlPicker uses different models for storing the data and when converted in a PropertyValueConverter.

This model is used when storing the data and it has a QueryString property
https://github.com/umbraco/Umbraco-CMS/blob/f3ec66be4c941c85afc081c51f6bb9b48a610379/src/Umbraco.Web/PropertyEditors/MultiUrlPickerValueEditor.cs#L159-L176 and the ProperteValueConverter concats the QueryString value to the Url from https://github.com/umbraco/Umbraco-CMS/blob/f3ec66be4c941c85afc081c51f6bb9b48a610379/src/Umbraco.Web/PropertyEditors/ValueConverters/MultiUrlPickerValueConverter.cs#L76-L83

**Comment by nul800sebastiaan** [comment]
Thanks @AreYouSureYouKen, do you have more steps to reproduce?

**Comment by AreYouSureYouKen** [comment]
Thank you @rasmusjp for linking the PropertyValueConverter! It seems it is something we're doing wrong and I have found the error we made. This issue can be closed.

As a small explanation, we're using the Multi Link Picker as a parameter editor for macro's as well. However that doesn't use PropertyValueConverters. Instead, it sends the data as json to the macro partial so we deserialize it from there. Which bypasses the functionality the PVC supplies, thus throwing the querystring data away.
